QA.md

```diff
@@ -1,186 +1,191 @@
-  - on Linux (AppImage)
-    1. `chmod +x <PATH_TO_DOWNLOAD>/radicle-upstream-X.X.X.AppImage`
-    2. Run `PATH_TO_DOWNLOAD>/radicle-upstream-X.X.X.AppImage` by executing it
-       from the terminal or clicking on it.
-
-### Packaging and distribution
-
-- [ ] Check that unreleased features are not visible in the UI
-  - [ ] Experimental features and developer helpers are not accessible
-    - Issues and Revisions tabs on the Project screen
-    - Wallets tab on the User Profile screen
-    - Design Sytem Guide is not listed in the shortcuts modal <kbd>?</kbd> and
-      the respective global hotkey is disabled <kbd>⌘</kbd>+<kbd>d</kbd>
-    - Tags are not visible in the revision selector on the Project Source screen
-- [ ] App icon is shown correctly
-  - [ ] macOS: dock, <kbd>⌘</kbd> + <kbd>tab</kbd> task switcher, mounted dmg,
-        app icon, "About radicle-upstream" window
-  - [ ] Linux: dock, menu bar
-- [ ] Version in the Settings screen matches version in package filename
+- [ ] Start Upstream
+  - on Linux: run `PATH_TO_DOWNLOAD>/radicle-upstream-X.X.X.AppImage` by
+    executing it from the terminal or clicking on it
+  - on macOS: run `/Applications/Radicle Upstream.app` by double clicking it
+- [ ] Onboarding starts on the _Get started_ screen
+- [ ] Pressing <kbd>enter</kbd> leads to the next screen
+- [ ] Handle validations work
+- [ ] Passphrase validations work
+- [ ] On the _All set!_ screen clicking the _Device ID_ copies it to the
+      clipboard
+- [ ] Clicking the _Go to my projects_ button leads to the _Profile_ screen
+- [ ] Hovering the network icon in the sidebar shows
+      _You're connected to 1 peer_
 
 
-### Onboarding
+### Creating projects
 
-- [ ] Always shown when no identity is set up
-  - [ ] First app start
-  - [ ] Subsequent app starts if the identity creation was not completed
-  - [ ] Can't exit onboarding before identity is created and global keyboard
-        shortcuts are disabled
-- [ ] It's possible to go to the next screen by pressing <kbd>enter</kbd> if
-      the input field validations allow it
-- [ ] Handle and passphrase validations work
-  - handle
-    - starts with a letter or digit
-    - is at least 2 characters long
-    - can contain the letters `a`-`z`, `A`-`Z`, digits `0`-`9` and special
-      characters `-`, `_`
-    - all of the following examples should show a validation error:
-      `_john`, `x`, `linu$$$`, `-moira`
-  - passphrase
-    - is at least 4 characters long
-    - the repeated passphrase is equal to the first one
-    - all of the following example pairs should show a validation error:
-      `1`/`1`, `12`/`12`, `123`/`123`, `supersecret`/`supersceret`
-- [ ] After passphrase input the identity is created and a copyable URN is
-      provided
-- [ ] After completion we land on our profile page which contains a placeholder
-      with instructions on how to create your first project
+- [ ] Can create a new project with a new repository
+  - [ ] Validations work
```

Comment:
Would be good to list here examples of invalid inputs. At the same time, this is all UI-tested so ya...

Reply:
Yea, we had those (see a couple of lines above), I decided to remove them because they're all covered by our cypress test suite and they took quite a while to check manually.

My thinking here is that this is just a very simple smoke test that's fast to execute. Like when you delete the input contents it shows the validation, nothing more.

Reply:
Yeah I am aware of that, what I meant to say is that maybe having an example of an invalid input instead of "validation work" would be better after all since both want to test the same but the latter makes it implicit what to test for.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1485/commits/faf154b3a93a3bd007a95c23ce09dd471669e122